contracts/tasks/e2eCron.ts

```diff
@@ -0,0 +1,112 @@
+require('dotenv').config()
+
+import {spawn} from 'child_process';
+
+const TIMEOUT_SECONDS: number = 60 * 60
+
+async function main(): Promise<void> {
+  const command = 'npx';
+  const contractAddress = process.env.TEST_CONTRACT_ADDRESS
+  const oracleAddress = process.env.TEST_ORACLE_ADDRESS
+  const network = process.env.TEST_NETWORK
+  const slackWebHookUrl = process.env.TEST_SLACK_WEBHOOK_URL
+  if (!contractAddress) {
+    console.log(".env is missing TEST_CONTRACT_ADDRESS")
+    return
+  }
+  if (!oracleAddress) {
+    console.log(".env is missing ORACLE_ADDRESS")
+    return
+  }
+  if (!network) {
+    console.log(".env is missing TEST_NETWORK")
+    return
+  }
+  if (!slackWebHookUrl) {
+    console.log(".env is missing TEST_SLACK_WEBHOOK_URL")
+    return
+  }
+  const args: string[] = [
+    "hardhat",
+    "e2e",
+    "--contract-address", contractAddress,
+    "--oracle-address", oracleAddress,
+    "--network", network
+  ];
+
+  while (true) {
+    console.log(`Running e2e tests with `)
+    await runTests(command, args, slackWebHookUrl)
+    console.log(`Run done, sleeping for ${TIMEOUT_SECONDS} seconds`)
+    await new Promise((resolve) => setTimeout(resolve, TIMEOUT_SECONDS * 1000));
+  }
+}
+
+async function runTests(
+  command: string,
+  args: string[],
+  slackWebHookUrl: string,
+): Promise<void> {
+  try {
+    await new Promise((resolve, reject) => {
+      const childProcess = spawn(command, args, {shell: true});
```

Comment:
This is necessary because Promise.race does not mean that the slower Promise is stopped..
Spawning a new process just to ensure we dont leave any promises dangling in the tests

Reply:
Wait but if starting a new process does the previous gets stopped?